crates/rolldown/tests/esbuild/default/var_relocating_bundle/_config.json

```diff
@@ -21,7 +21,6 @@
         "name": "function-nested_js",
         "import": "function-nested.js"
       }
-    ],
-    "treeshake": false
```

Comment:
esbuild did not disable tree shake in this suite

Reply:
https://github.com/evanw/esbuild/blob/d34e79e2a998c21bb71d57b92b0017ca11756912/internal/bundler_tests/bundler_default_test.go#L6333-L6344